config/packages/doctrine.yaml

```diff
@@ -8,15 +8,35 @@ doctrine:
             charset: utf8mb4
             collate: utf8mb4_unicode_ci
 
-        url: '%env(resolve:DATABASE_URL)%'
+        url: '%database_url%'
     orm:
-        auto_generate_proxy_classes: true
-        naming_strategy: doctrine.orm.naming_strategy.underscore
+        report_fields_where_declared: true
+        auto_generate_proxy_classes: false
```

Comment:
I think the Doctrine rector rule might have done this. Otherwise it was upgrade by a recipe or by yours truely. 

As you can tell. I dont remember myself :P In my defence, this code was changed at July last year.